### DISCUSSION TITLE
Multi-domain in single docker

### DISCUSSION THREAD

**Discussion by yodaphone** [rustdesk/rustdesk#1215] [Q&A]
I wanted to know if we can have multiple domains served by a single server?

like for e.g.

command: hbbs -r mdm.foo1.com:21117 -k 12345safjdhsdjkgh
command: hbbs -r mdm.foo2.com:21117 -k 123456safjdhsdjkgh

**Comment by chiehw** [comment]
Associate hbbs with IP, IP can bind multiple domain.
